modules/govuk/manifests/apps/ckan/cronjobs.pp

```diff
@@ -0,0 +1,19 @@
+# == Class: govuk::apps::ckan::cronjobs
+#
+class govuk::apps::ckan::cronjobs {
+
+  govuk::apps::ckan::paster_cronjob { 'harvester':
```

Comment:
Do we need the `harvester clean_harvest_log` task too? It's documented – https://github.com/ckan/ckanext-harvest#setting-up-the-harvesters-on-a-production-server